index.json

```diff
@@ -3,5 +3,6 @@
   "aws/microservice-ecommerce",
```

Comment:
Can you remove `index.json` from the PR? We have created a script to automatically generate it, see [LOVE-107](https://xebialabs.atlassian.net/browse/LOVE-701).

Reply:
Yep I'll do that